### PR TITLE
Table

### PR DESCRIPTION
**Pull request by marcheliks**
DSM - 13 (Papers)
DSM - 14 (Students)
Table

### CODE REVIEW

File: web/Table/Table.html [hunk 1]
```diff
@@ -0,0 +1,304 @@
...
+      <td></td>
+      <td></td>
+      <td></td>
+      <td></td>
+   
+  </tbody>
+</table>
+
+</div>
+</html>
```
Comment (vdzjubina) [inline web/Table/Table.html:304]: HTML file should be in the `views` folder.

File: web/Table/Table.css [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+
+input[type=text], select {
+    width: 80%;
+    padding: 12px 20px;
+    margin: 8px 0;
+    display: inline-block;
+    border: 1px solid #ccc;
+    border-radius: 4px;
+    box-sizing: border-box;
+}
```
Comment (vdzjubina) [inline web/Table/Table.css:87]: Please use Bootstrap in your HTML file instead of a CSS file.

File: web/Table/Table.html (hunk 1 above)
Comment (Kalituha21) [reply web/Table/Table.html:304]: Not resolved, but we need the code to be able to complete other tasks, so merging

Comment (Kalituha21) [reply web/Table/Table.html:304]: Not resolved, but we need the code to be able to complete other tasks, so merging

File: web/Table/Table.css (hunk 2 above)
Comment (Kalituha21) [reply web/Table/Table.css:87]: Not resolved, but we need the code to be able to complete other tasks, so merging

Comment (Kalituha21) [reply web/Table/Table.css:87]: Not resolved, but we need the code to be able to complete other tasks, so merging

**Review by Kalituha21 — APPROVED** [review]
Bad. Mistakes not fixed. Code won't work in production

**Pull request merged**